pytorch_blade/pytorch_blade/torch-mlir/lib/Dialect/TorchConversion/Transforms/DiscDecomposeComplexOps.cpp

```diff
@@ -89,26 +89,28 @@ LogicalResult ConvertAtenOp<OperatorOp>::matchAndRewrite(
     ConversionPatternRewriter& rewriter) const {
   Location loc = op.getLoc();
   auto name = op.name();
-  if (std::string("aten._autocast_to_reduced_precision") == name) {
+  if (std::string("aten._autocast_to_reduced_precision") == name ||
```

Comment:
nit: Really need to create a temporary std::string variable for comparison? will `name.equal("xxxx")` work?

Reply:
nice suggestion!